### PR TITLE
Router.createModelRouter() (for #16)

### PR DESCRIPTION
**Pull request by KeithWoods**
Work in progress, PR for reviews

### CODE REVIEW

**Comment by KeithWoods** [comment]
This is ready for review and merge

File: ReadMe.md [hunk 1]
```diff
@@ -914,6 +914,40 @@ Sending results event for StaticDataB
 Adding static data [StaticDataB] to model
 ```
 
+## Model Specific Routers
+Sometimes it can be too verbose to provide a model's id.
+Perhaps you have a subsystem that only ever deals wih a single model or your app is small so only warrants a single model.
+In these scenarios you can use `router.createModelRouter(modelId)` to return a `ModelRouter` which proxies `router` and passes the correct `modelId`.
+The modelId still exists and is required by the underlying `Router` however the object returned from 'createModelRouter()' will deal with passing it.
+This simplifies the API for use by the application and/or sub system.
```
Comment (raybooysen) [inline ReadMe.md]: "subsystem", not "sub system"

**Comment by raybooysen** [comment]
Looks good!

**Comment by KeithWoods** [comment]
Cheers Ray. Pushed the spelling change. I'll merge this an push to npm

**Pull request merged**